### DISCUSSION TITLE
Dero astrobwt/v3 support

### DISCUSSION THREAD

**Discussion by Knevil** [Lolliedieb/lolMiner-releases#1805] [Ideas]
@Lolliedieb, any thoughts on developing a GPU miner for Dero's v3?

**Comment by Cybertiron** [comment]
Is dero not cpu?

**Reply by Knevil** [reply]
currently yes, but a GPU miner is wanted.

**Reply by Lolliedieb** [reply]
No and it will not happen. 

I have sunken about 3 weeks of work to make a miner for AstroBwtV2 which was planned to be released with 1.52 ...
But just a week before the planned release it was forked to AstroBwtV3 in an extremely spontaneous action - less then half a day between code release and fork with a fix release just two hours before the fork...

Reason for this was to get rid of fpgas as soon as possible, because they were afraid they have taken over the network. But: for a healthy network that would not even be an issue - an fpga secured network would even be more secure then a GPU one. But that is not the point.... 

The network has some fundamental flaws that I would like to touch on here: 
First, it is not at all resistant to a lot of hashrate being united on one miner. If a node has too much hashrate, it can selfishly mine and place more of its own miniblocks than it should - miniblocks from other miners are then orphaned. This is also the reason why you have such a problem with FPGAs. 
Secondly, due to the short block time, there are many orphans - and no system to prevent them effectively. Look at kaspa: with even shorter block time there is no problem.
Third, the time in which the first nine miniblocks are mined is much more profitable than the remaining time in which a height is sealed. But only this time counts for the network diff - which makes the network vulnerable to certain attacks.  

These bugs were all known at the time and if they had been fixed, FPGAs might not have been such a problem. But instead of fixing them, they instead felt through the hasty hardfork, which in summary is just idiotic.
Because the new algorithm is not worse suited for FPGAs than the old one - only one would have to write a bitstream. Furthermore, the whole GPU mining community, which the coin had, was completely blindsided - since there was no GPU miner, we had to deal with a complicated CPU-only coin, for which there were soon optimized miners, but which were not public at first. Furthermore: everyone who wanted to run a Dero node had to react and update within 24 hours, otherwise the node stopped at fork. Is that any way to deal with your community? How will you ever reach the mass market with such an update policy?

In addition, any kind of criticism from some discord moderators of dero is considered an insult to the king. They have a very short fuse. Dero is perfect for these people and any problems that occur are just stupid users or hackers trying to test the system....

I'm sorry to say this, but until they fix their problems and rethink their whole strategy on how to deal with bugs and the community, I can't recommend anyone to put money, time or work or anything else worthwhile into this project....

**Reply by Knevil** [reply]
thanks for the reply and explanation. hopefully they straighten all this up
